multiversx_sdk_core/address.py

```diff
@@ -36,20 +36,38 @@ def new_from_bech32(cls, value: str) -> 'Address':
         hrp, pubkey = _decode_bech32(value)
         return cls(pubkey, hrp)
 
+    @classmethod
+    def from_bech32(cls, value: str) -> 'Address':
+        """The `from_bech32()` method is deprecated. Please use `new_from_bech32()` instead"""
```

Comment:
should we use a library for deprecated items? for example: https://deprecation.readthedocs.io/en/latest/

Reply:
since this is the core package and we want to have as less dependencies as possible, I don't think we should use a package for that.